scripts/clean_osm_data.py

```diff
@@ -192,6 +194,28 @@ def filter_voltage(df, threshold_voltage=35000):
     return df
 
 
+def filter_links(df):
+
+    # convert frequency to int
+    df.loc[:, "tag_frequency"] = df["tag_frequency"].astype(int)
+
+    # TODO Fix SettingWithCopyWarning, details via https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
+    # keep only lines with zero frequency as the most useful indication of DC lines
+    df = df[df.tag_frequency == 0]
+
+    return df
+
+
+def filter_ac(df):
```

Comment:
@ekatef I'd recommend to use "df.tag_frequency != 0" to find ac lines, which is more general (e.g. US)

Reply:
@davide-f, thank you!
Yes, I have also thought that 50Hz default used for AC could be generalised. Not sure however if  `df.tag_frequency != 0` would be a good AC-indicator as frequency for AC lines is missed in OSM data rather often. Probably, it would make sense to extract the AC-frequency values for each country to be used instead 50Hz across the given country?

Reply:
AC lines at 50Hz or 60Hz or other values are still AC lines and at first approximation, we can consider them as they were exactly the same (in practice there are obviously differences but negligible for our needs).
So where tag_frequency is specified and it is 0, I'd say they are DC lines, all the rest are AC regardless of the value of tag_frequency (nan, 50, 60, etc).

Reply:
That absolutely makes sense. 
I have changed filtering here and added TODOs to the points where 50Hz values are used.